### PR TITLE
Fixed #26985 -- Doc'd ForeignKey.to_field bonds

### PR DESCRIPTION
**Pull request by joncasdam**
Make explicit in the documentation the requirement for the field referenced at ForeignKey.to_field to be `unique=True`.

https://code.djangoproject.com/ticket/26985

### CODE REVIEW

File: docs/ref/models/fields.txt [hunk 1]
```diff
@@ -1424,7 +1424,8 @@ The possible values for :attr:`~ForeignKey.on_delete` are found in
 .. attribute:: ForeignKey.to_field
 
     The field on the related object that the relation is to. By default, Django
-    uses the primary key of the related object.
+    uses the primary key of the related object, therefore ensure the referenced
```
Comment (audiolion) [inline docs/ref/models/fields.txt]: I would probably phrase it:

`uses the primary key of the related object, therefore ensuring the referenced field...

Comment (joncasdam) [inline docs/ref/models/fields.txt]: I see, but the idea is to keep clear that if your not using the default from Django (referencing to a pk field), you might then use a field that is `unique=True`.
So I think I'd put as `therefore be sure you're referencing a field that is unique=True in order to stay aligned with the design`.

Comment (audiolion) [inline docs/ref/models/fields.txt]: yeah that is more clear

**Comment by joncasdam** [comment]
@audiolion commit amended.

**Comment by timgraham** [comment]
merged in 9da45ff352db1d1b7c64e5c724e9b0b31822b967, thanks!

**Pull request closed without merging**